blog/content/post/2021-10-10-optiplex-dasharo.md

```diff
@@ -0,0 +1,173 @@
+---
+title: 'Dasharo for Dell OptiPlex 7010 / 9010'
+abstract: 'Open source firmware may be hundred times better than the
+           proprietary one. On the example of Dell OptiPlex 7010 / 9010 we will
+           show you the advantages of Dasharo firmware on this machine.'
+cover: /covers/dasharo-sygnet.svg
+author: michal.zygowski
+layout: post
+published: true
+date: 2021-10-10
+archives: "2021"
+
+tags:
+  - firmware
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+Dell OptiPlex 7010 / 9010 is the majority of machines used in 3mdeb office by the
+engineers and developers. It is only natural for a company propagating open
+source firmware (OSFW) to utilize their own product in daily work. The process
+of porting coreboot on the machine took significant amount of free time (which
+you may read about in [other blog posts](https://blog.3mdeb.com/2020/2020-06-24-dell-optiplex-port/)
+by the way) but it was definitely worth it. It opened a path for developing a
+firmware that would squeeze out the full potential of the platform. You may be
+wondering what else a platform may do that OEM didn't implement. But I will
+tell you this, with open source firmware almost anything is possible and with
+[Dasharo firmware](https://dasharo.com/) we make it a reality.
+
+## ME neutralization
+
+Intel Management Engine is a very controversial component of modern Intel
+chipsets and SoCs due to its capabilities and features. One of the most fearful
+features is the remote management and connectivity. Intel ME is able to utilize
+network via dedicated network controller on vPro machines without any awareness
+of the operating system software. It is used for software like Intel Active
+Management Technology ([AMT](https://www.intel.co.uk/content/www/uk/en/architecture-and-technology/intel-active-management-technology.html))
+or [Computrace](https://i.dell.com/sites/content/business/solutions/brochures/en/Documents/absolute-overview.pdf).
+Such features and capabilities often raises doubt and concerns about privacy.
+Moreover it is a few megabytes of proprietary closed source firmware of unknown
+quality with uncertain other impacts on the system. Many people often want to
+disable ME or even neutralize it to reduce the attack surface. This is what we
+offer with Dasharo firmware on Dell OptiPlex 7010 / 9010, [me_cleaner](https://github.com/corna/me_cleaner)
+is being applied on the firmware image which can be flashed on the board. From
+a 6MB of unknown firmware only dozens of kilobytes are left.
```

Comment:
I think referring this would be great: https://osfw.slack.com/archives/C9ZLS0U4F/p1633701873113300

Reply:
Added two sentences. @pietrushnic Please see if this is what you had in mind

Reply:
@miczyg1 I think last sentence you added that blog posts are contradicting should be changed. In EU we have clear article talking about decompilation (article 6): https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32009L0024

Finally you can say IANAL

Reply:
Done, last sentence removed and added reference to the decompilation article.